essential-projects/datasource_adapter_localstorage
# ✨Implement Local Storage Adapter

## What did you change?

Up until now the localstorage-adapter was just a mock that didn't actually do much. This branch implements all the necessary functionality so that the localstorage-adapter can be used in place of the postgres-adapter

## How can others test the changes?

Replace the postgres-adapter with the localstorage-adapter in your integration-project, update your config accordingly and see how everything still works.

An example is in the skeleton-repository, see https://github.com/process-engine/skeleton/pull/37

## PR-Checklist

Please check the boxes in this list after submitting your PR:
